summary_workflows/identification/I2_benchmark/I2_benchmark_specification.md

```diff
@@ -47,45 +64,146 @@ Fields:
+
+Input data:
+
+- I2: RNA-Seq data compared with 3'end sequencing data
+- I3: RNA-Seq data compared with PAS databases
+- I5: Simulated RNA-Seq data
+
+
+Ranking: The best performing tool is the one with the highest precision value.
+
+2. **bar plot** visualizing **Sensitivity** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - benchmarked tool  
+**Y axis** - sensitivity
+
+Input data: 
+
+- I2: RNA-Seq data compared with 3'end sequencing data
+
+
+Ranking: The best performing tool is the one with the highest sensitivity value.
+
+3. **2D scatter plot** visualizing **TPR and FPR** of poly(A) site identification. Separate plots should be prepared for different values of distance threshold:
+
+- 10 nt
+- 50 nt
+- 100 nt
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_
+
+Input data:
+
+- I2: RNA-Seq data compared with 3'end sequencing data
+- I3: RNA-Seq data compared with PAS databases
+- I5: Simulated RNA-Seq data
+
+Ranking: The best performing tool is the one with the highest TPR combined with lowest FPR (top left part of the plot) and the worst performing tool is the one with the lowest TPR combined with highest FPR (bottom left part of the plot. The plot should be divided into diagonal quartiles based on the distance from optimal performance. Alternatively, if the plot is divided into square quartiles, the following ranking order should be applied: top-left, top-right, bottom-left, bottom-right.
+
+4. **bar plot** visualizing **AUC** of ROC plot with single AUC value for each tool.
+
+**X axis** - benchmarked tool  
+**Y axis** - AUC
+
+Input data:
+
+- I2: RNA-Seq data compared with 3'end sequencing data
+
+
+Ranking: The best performing tool is the one with the highest AUC value.
+
+Optional plots:
+
+5. **2D line plot** visualising **TPR and FPR** in the form of [ROC curve][roc-curve], i.e. TPR and FPR calculated for a range of distance thresholds are plotted together. 
+
+**X axis** - FPR(d)  
+**Y axis** - TPR(d)  
+where _d - distance threshold_ in range 0-100 nt with 10 nt increments.
+
+Input data:
+
+- I2: RNA-Seq data compared with 3'end sequencing data
+
+Ranking: For each tool, the area under the curve is calculated. The best performing tool is the one with the highest AUC.
+
+Note: 2D line plot is not supported in OpenEBench yet. If it's not implemented, the data should be visualised outside of OpenEBench.
+
+
 ## Outputs
 
-| # | Format | Link | Example data |
-  | --- | --- | --- | --- |
-  | 1 | JSON | [Specification][spec-json] | [Link][out1] |
-  
-### Additional info outputs
+Calculated metrics are saved in JSON file adhering to OpenEBench schema. 
+Assessment output contains values of calculated metrics for a given input dataset.
+Consolidation output contains summarized data in a format suitable for plotting, e.g. single values for barplot or X,Y value pairs for 2D scatter plot.
```

Comment:
Maybe: "Consolidation output contains summarized data *from all benchmarked tools within one challenge* in a format suitable for plotting..."